### PR TITLE
Feature/uservotes

### PR DESCRIPTION
**Pull request by F-Kirchhoff**
- refactor voting system on proposals to use the user sessions to recognize users
- adapt proposal card so users can vote directly from it via 4 buttons

### CODE REVIEW

File: src/app/pages/TopicView/TopicView.tsx [hunk 1]
```diff
@@ -37,12 +39,17 @@ function TopicView(): JSX.Element {
     boardName,
     topicId,
   })
+  const [_vote, fetchVote] = useFetch<Vote>(`/api/votes`, {
```
Comment (julialoeschel) [inline src/app/pages/TopicView/TopicView.tsx:42]: why is this _vote?

File: src/server/routes/users.ts [hunk 2]
```diff
@@ -7,7 +7,9 @@ const users = express.Router()
 
 users.get('/', async (req: Request, res: Response) => {
   if (!req.session || !req.session.user) {
-    res.status(400).send()
+    console.log('Bad Session!')
```
Comment (julialoeschel) [inline src/server/routes/users.ts:10]: 🕵️

File: src/server/routes/users.ts [hunk 3]
```diff
@@ -17,7 +19,9 @@ users.get('/', async (req: Request, res: Response) => {
   const user = await users.findOne({ 'public.username': username })
 
   if (!user) {
-    res.status(404).send(`Error: no user called ${username} found.`)
+    console.log('No User!')
```
Comment (julialoeschel) [inline src/server/routes/users.ts:22]: 🕵️

File: src/server/routes/users.ts [hunk 4]
```diff
@@ -17,7 +19,9 @@ users.get('/', async (req: Request, res: Response) => {
...
+
+    res.status(404).send(null)
```
Comment (julialoeschel) [inline src/server/routes/users.ts:24]: do you need the `null`?

**Review by julialoeschel — COMMENTED** [review]
great work! 🐙

**Review by derjubu — APPROVED** [review]
LGTM 🐼

**Pull request merged**